nrpy/infrastructures/BHaH/cmdline_input_and_parfiles.py

```diff
@@ -287,7 +287,7 @@ def register_CFunction_cmdline_input_and_parfile_parser(
 """
     if found_REAL:
         prefunc += r"""
-static void read_double(const char *value, double *result, const char *param_name) {
+static void read_double(const char *value, REAL *result, const char *param_name) {
```

Comment:
This was a patch that I wanted to discuss in the case `REAL != double`.  In principle this is simply reading data from the par file so precision may not be so important.  In principle `read_double` is still a correct function name since it uses `strtod(value, &endptr);` , however, the read-in value gets cast to whatever `REAL` is defined as which is potentially undesirable.  

Alternatively, a separate function would need to be declared based on `fp_type` with appropriate naming and standard C function for reading the data which would increase the logical branching here.